packages/legend-studio/src/stores/ViewerStore.tsx

```diff
@@ -152,7 +153,9 @@ export class ViewerStore {
           `Can't have both version ID and revision ID specified for viewer mode`,
         );
       }
+
       let entities: Entity[] = [];
+      let projectConfiguration: PlainObject<ProjectConfiguration>;
```

Comment:
can we name this `rawProjectConfiguration`
and the serialized one `projectConfiguration`
thats more inline with what we do in the rest of the app